test/range/a11y-spec.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import ReactTestUtils from 'react-dom/test-utils';
+import Enzyme, { mount, shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import simulateEvent from 'simulate-event';
+import assert from 'power-assert';
+import sinon from 'sinon';
+import Range from '../../src/range/index';
+import { unmount, testReact } from '../util/a11y/validate';
+
+/* eslint-disable react/no-multi-comp */
+Enzyme.configure({ adapter: new Adapter() });
+
+describe('Range A11y', () => {
+    let wrapper;
+
+    afterEach(() => {
+        if (wrapper) {
+            wrapper.unmount();
+            wrapper = null;
+        }
+        unmount();
+    });
+    it('should not have any violations for range without tips', async () => {
+        wrapper = await testReact(<div>
+            <Range defaultValue={10} hasTip={false} tooltipVisible={false} />
+        </div>);
+        return wrapper;
+    });
+    it.skip('should not have any violations for range with tips', async () => {
```

Comment:
can you add a comment for why this is skipped?

Reply:
Done